gffquant/handle_args.py

```
@@ -58,24 +65,26 @@ def validate_args(args):
+
+    if args.strand_specific:
+        raise NotImplementedError("Strand-specific counting is currently disabled.")
+
     if args.restrict_metrics:
-        restrict_metrics = set(args.restrict_metrics.split(","))
-        invalid = restrict_metrics.difference(('raw', 'lnorm', 'scaled', 'rpkm'))
-        if invalid:
-            raise ValueError(f"Invalid column(s) in `--restrict_metrics`: {str(invalid)}")
-        args.restrict_metrics = tuple(restrict_metrics)
+        args.restrict_metrics = None
+        warnings.warn("Metric restriction is currently ignored.")
```

Ruff (0.16.0)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)